lib/test/test-panel.js

```diff
@@ -14,6 +19,36 @@ export default class TestPanel extends EtchComponent {
     if (this.props.model) {
       this.props.model.view = this
     }
+
+    this.locationRegex = /([\w-/.\\:]*.go:\d+)/g
```

Comment:
this does not capture a `<file>:<line>:<column>` like `foo/bar.go:23:4`
tbh: I haven't seen any output that contained this file syntax, so feel free to ignore this comment